js/views/SurveyDetailsPage.js

```diff
@@ -81,7 +111,6 @@ const SurveyDetailsPage = React.createClass ({
       survey.status = 'declined'
     })
     this.props.navigator.pop()
-    // this.props.navigator.push({path: 'surveylist', title: 'Surveys', sceneConfig: 'FloatFromLeft'})
   },
 
   showForms(tab) {
```

Comment:
@jgoley renderFormButtons method isn't used as well due to having a single trigger type, I'll add a note in pivotal so we don't forget this.  IMO its confusing leaving so much dead code in place.